cdk/visit/lambda_code/log_visit.py

```diff
@@ -130,6 +132,11 @@ def handler(request, context):
     try:
         # Get the username from the request body.
         username = json.loads(request["body"])["username"]
+        location = ""
+        try:
+            location = json.loads(request["body"])["location"]
+        except Exception as e:
+            print(e)
```

Comment:
It would be better to use Lambda's logging library instead. Check out this article that explains how & why: https://towardsdatascience.com/why-you-should-never-ever-print-in-a-lambda-function-f997d684a705